.lintstagedrc.json

```diff
@@ -0,0 +1,11 @@
+{
+    "*": [
+        "bash -c 'codespell'"
```

Comment:
You may have explained to me already but wondering why this is wrapped in `bash -c`? I changed it locally to just `codespell` and it seemed to work as expected still.

Reply:
was needed before the bash pieces we put in the pre-commit. no longer needed.